package.json

```diff
@@ -52,6 +52,7 @@
     "swagger-express-mw": "^0.7.0",
     "swagger-tools": "^0.10.4",
     "translate-api": "^0.3.18",
+    "url-join": "^4.0.1",
```

Comment:
You might want to remove `package-lock.json` and re-run the command using `yarn`.